README.md

```diff
@@ -22,40 +31,50 @@ heroku buildpacks:add --index 1 https://github.com/DataDog/heroku-buildpack-data
 heroku config:set DD_API_KEY=<your API key> # note: older releases called this DATADOG_API_KEY
 heroku config:set DD_SERVICE_NAME=<your service name>
 heroku config:set DD_SERVICE_ENV=<your service env>
-heroku config:set DD_HOSTNAME=<your service name> # optional, defaults to dyno hostname
 
 git push heroku master
 ```
 
-You can create/retrieve the `DD_API_KEY` from your account on [this page](https://app.datadoghq.com/account/settings#api).
-API Key, not application key.
+Once complete, the Datadog agent and Datadog Trace agent will be started automatically with the Dyno startup.
+
+The Datadog agent provides a listening port on 8125 for statsd/dogstatsd metrics and events. Traces are collected on port 7777 by the Datadog Trace agent, then information is forwarded on to the Datadog Agent.
+
+### Configuration
+
+In addition to the environment variables shown above, there are a number of others you can set:
+
+| Setting | Description|
+| --- | --- |
+| DD_API_KEY | *Required.* Your API key is available from [the Datadog API Integrations page](https://app.datadoghq.com/account/settings#api). Note that this is the *API* key, not the application key. |
+| DD_SERVICE_NAME | *Optional.* Sets the service name for the traced application. For more information, about naming services, see the [Datadog Tracing Docs](https://app.datadoghq.com/trace/docs/tutorials/terminology). |
```

Comment:
@talwai `DD_SERVICE_NAME` was added to the example during the last PR merge. I didn't see anything in `extras/run-dogstatsd.sh` that adds it to the datadog.conf. Is this something the agent reads from the environment or is it something the language specific libraries read?